source/accessible-media/playing.html.erb.md

```diff
@@ -18,29 +18,31 @@ support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/
 
 ### Desktop-browser media players
 
-All modern Web browsers will provide default media players for embedded
-media when they detect the `video` or `audio` elements, and all will
-render timed text delivered via the `track` element. All browsers will
+All modern web browsers will provide default media players for embedded
+media when the `<video>` or `<audio>` elements are used, and all will
+render timed text delivered via the `<track>` element.
+
+All browsers will
 automatically surface a caption toggle in the player control bar if a
-caption track is detected. However, note that each browser will present
+caption track is detected. Note that each browser will present
 not only a different UI, but also a different default style for
```

Comment:
This sounds like a negative, however having a similar experience in the browser of choice seems better to me than every site doing their own UI.